.vscode/settings.json

```diff
@@ -0,0 +1,48 @@
+{
+  "explorer.fileNesting.enabled": true,
+  "explorer.fileNesting.expand": false,
+  "explorer.fileNesting.patterns": {
+    ".prettierrc.json": ".*, *.md, LICENSE"
+  },
+
+  // Hides directories/files from sidebar.
+  // Excludes them from search by default, as well as from quick open.
+  "files.exclude": {
```

Comment:
Is it possible to exclude from search but not from sidebar? If not, please drop.

Reply:
Yes, it is possible, but how often do you use VSCode to access these directories...?